docs/static/transforming-data.asciidoc

```diff
@@ -554,6 +580,23 @@ filter {
 }
 --------------------------------------------------------------------------------
 
+<<plugins-filters-memcached,memcached filter>>::
+
+Enables key/value lookup enrichment against a Memcached object caching system.
+It supports both read (GET) and write (SET) operations. It is a notable addition
+for security analytics use cases. For example, you can use this plugin to query
+for a value, and set it if not found.
++
+[source,txt]
+-----
+filter {
+  memcached {
+    url => "http://example.com"
+    verb => GET
+    body => {TODO-complete example
```

Comment:
TO DO:  Complete the cache misses sample

Reply:
@karenzone I'm not sure whether we need the cache misses example here on this page as its more of a high level overview of all lookup enrichment capabilities. I think just noting it in the memcached filter plugin docs is likely sufficient.

Reply:
WIP to move and flesh out example in memcached plugin docs.